include/deal.II/lac/precondition.h

```diff
@@ -877,7 +1056,9 @@ PreconditionRichardson::AdditionalData::AdditionalData (
 inline
 PreconditionRichardson::PreconditionRichardson ()
   :
-  relaxation(0)
+  relaxation(0),
+  _m (0),
+  _n (0)
```

Comment:
I'm not fond of `_name` variables. Can't we just name them `n_row` and `n_columns` or similar?